tests/performance_tests_doolittle.py

```diff
@@ -0,0 +1,15 @@
+import numpy as np
```

Comment:
tests filename should start with test_
I'd suggest renaming to test_doolittle_factorization_performance